src/components/AttachmentPicker/index.native.js

```diff
@@ -28,54 +35,76 @@ const documentPickerOptions = {
 };
 
 /**
- * Launch the DocumentPicker. Results are in same format as ImagePicker, so we can pass the repsonse to the
- * callback as is.
+ * Inform the users when they need to grant camera access and guide them to settings
+ */
+function showPermissionsAlert() {
+    Alert.alert(
+        'Expensify.cash does not have access to your camera.',
+        'Please enable the permission and try again.',
+        [
+            {
+                text: 'Cancel',
+                style: 'cancel',
+            },
+            {
+                text: 'Settings',
+                onPress: () => Linking.openSettings(),
+            },
+        ],
+        {cancelable: false},
+    );
+}
+
+/**
+ * A generic handling when we don't know the exact reason for an error
+ *
+ */
+function showGeneralAlert() {
+    Alert.alert(
+        'Oops. An error occurred while selecting an attachment.',
+        'Please try again.',
```

Comment:
Title should be brief:

Title:  Attachment Error
Message: An error occurred while selecting an attachment, please try again